### ISSUE TITLE
tax_id check sum amt problem, PreProd, #J359

### ISSUE THREAD

**Issue by ehealthua-tech** [edenlabllc/ehealth.api#3185]
Проблема з ІПН:
"Частина пацієнтів мають діючи ІПН із невірною контрольною сумою. Під час укладання декларацій валідація такі ІПН не пропускає. 
Наразі маємо декілька таких підтверджених звернень. 
За потреби можу надати фотокопію оригіналів довідок про присвоєння ІПН."
Більше інформації тут https://drive.google.com/drive/u/1/folders/1YycDy8skxmHmP-Vsb3NKtdt-4PyA6i7J?ogsrc=32
Додаткова інформація про контрольну суму:
Сума розраховується за певним алгоритмом. Проте деякі коди видані ДФС містять помилки. Перевірити коди можна тут https://igov.org.ua/subcategory/1/1/situation/13.

**Comment by svetavedmed** [comment]
да, пришлите пожалуйста фото этих инн личным сообщением. Спасибо.
